cpp_utils/include/cpp_utils/types/impl/cast.ipp

```diff
@@ -28,5 +28,12 @@ void* cast_to_void_ptr(
     return static_cast<void*>(const_cast<T*>(c));
 }
 
+template <typename T, typename U>
+bool can_cast(const U& el)
+{
+    const T* _ptr = dynamic_cast<const T*>(&el);
+    return _ptr;
+}
+
```

Comment:
I do not see the utility of these functions as this is something that should be exposed in the actual code. 

In any case, `can_cast` is not returning a `bool`. Intead, it crash with a bad_cast exception or return a nulptr

Reply:
1. It is true that this is normally in plain code. However, it is a bit verbose and difficult to understand (or at least, much more difficult than a funcion called can_cast).

2. This does not crash. dynamic_cast to a ptr does not throw an exception, returns a nullptr. And check a ptr to a boolean is to check if it is not a nullptr, so it actually returns what it must.